pkg/kubectl/resize.go

```diff
@@ -86,7 +112,7 @@ func (resize *ReplicationControllerResizer) Resize(namespace, name string, preco
 	controller.Spec.Replicas = int(newSize)
 	// TODO: do retry on 409 errors here?
 	if _, err := rc.Update(controller); err != nil {
-		return "", err
+		return "", ControllerUpdateError{controller.ResourceVersion, controller.Spec.Replicas, err}
```

Comment:
I think it's pretty confusing to put controller.Spec.Replicas into the error; it was set three lines above and will always be the thing that was passed in.